devtools/doc/rpc.py

```diff
@@ -102,7 +102,6 @@ def main():
     print("# CKB JSON-RPC Protocols")
     newline(2)
 
-    cases = sorted(cases, key = lambda x: x["module"] + x["method"])
```

Comment:
It requires the JSON must put the RPCs in the same module together. But I think it is a reasonable requirement.

Reply:
agreed